app/models/user.rb

```diff
@@ -1,10 +1,12 @@
 class User < ApplicationRecord
   has_secure_password
 
-  validates :name, presence: true, uniqueness: true
+  with_options presence: true do
+    validates :name
+    validates :email, allow_blank: true, uniqueness: true, format: { with: URI::MailTo::EMAIL_REGEXP }
```

Comment:
does it make sense to have both presence true and allow_blank true in the same validation?
Please follow what you did in product.rb everywhere